### ISSUE TITLE
Additional analyses?

### ISSUE THREAD

**Issue by fgregg** [sc3/cook-convictions-data#20]
What's the status of this project? Are you going to be doing additional analyses?

Here's some questions I think could have a real impact.
1. What is the distribution of the time between arrest and final disposition, broken down by charge. Speedy justice? For what types of charges do we need more capacity?
2. Do police districts vary in their conviction rates. We have arrest data from the data portal and now we have conviction data. Does the rate of convictions per arrest vary?
3. How do charges get pled down from initial charge to final charge?

**Comment by danxoneil** [comment]
Hi, Forest. Status is that we just launched it yesterday and we're definitely into doing additional analysis. Good questions, especially the first one, which is easily done with the data in hand. We're on this— thank you!

If there is arrest data on the data portal, that would be news to me. They certainly have crime reports, but I don't think they have arrest reports. We wrote about arrest data in Chicago on another project, Crime and Punishment in Chicago: http://crime-punishment.smartchicagoapps.org/arrest.html. Given the data, via FOIA, this could probably be determined.

On how charges are are pled down, Tracy Siska is probably the most knowledgable person on that— a blog post from him might shed light.

Other colleagues on the project might have other stuff to add. Keep the ideas coming!

**Comment by fgregg** [comment]
The crime data doesn't have arrest data, but the the crime reports do have
a field that indicate whether an arrest was made in connection to report
(the exact meaning of that field is unclear to me). For certain classes of
crime, like drug possession, it seems like the report is never really made
without an arrest.

For the plea down, don't you have the data in here: you have initial charge
and final charge, right?

On Wed, Dec 17, 2014 at 12:03 PM, danxoneil notifications@github.com
wrote:

> Hi, Forest. Status is that we just launched it yesterday and we're
> definitely into doing additional analysis. Good questions, especially the
> first one, which is easily done with the data in hand. We're on this— thank
> you!
> 
> If there is arrest data on the data portal, that would be news to me. They
> certainly have crime reports, but I don't think they have arrest reports.
> We wrote about arrest data in Chicago on another project, Crime and
> Punishment in Chicago:
> http://crime-punishment.smartchicagoapps.org/arrest.html. Given the data,
> via FOIA, this could probably be determined.
> 
> On how charges are are pled down, Tracy Siska is probably the most
> knowledgable person on that— a blog post from him might shed light.
> 
> Other colleagues on the project might have other stuff to add. Keep the
> ideas coming!
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/sc3/cook-convictions-data/issues/20#issuecomment-67364639
> .

## 

773.888.2718

**Comment by danxoneil** [comment]
Ah, I understand re: arrest data. And yes that field and the underlying data seems specious to me.

Re: plea down-- yes, have the data, for sure. Not sure it answers the question, "how" though. Classic expert analysis/ explication might help.

**Comment by ghing** [comment]
Yeah arrest date is sketchy.  In many cases, it's years before the disposition date.  I could never get a clear answer about the exact meaning of the field or the reason for the wide range of dates relative to other dates in the same record.

**Comment by ghing** [comment]
Oh, also from this data alone , I don't think we can map to police district because the address is the convicted person's stated home address, not the address of the crime/arrest.

**Comment by fgregg** [comment]
That's interesting about the arrest date.

On Wed, Dec 17, 2014 at 12:50 PM, Geoffrey Hing notifications@github.com
wrote:

> Oh, also from this data alone , I don't think we can map to police
> district because the address is the convicted person's stated home address,
> not the address of the crime/arrest.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/sc3/cook-convictions-data/issues/20#issuecomment-67371631
> .

## 

773.888.2718

**Comment by fgregg** [comment]
@ghing the date thing is the type of thing that twitter can help with.

**Comment by tjakester** [comment]
The answer is that we would need significant amount of data from the CPD that is just not available.  I created the Chicago Justice Project to fight for these types of data to be made public.  

The CPD incident data online details whether an arrest has been made, but it is a bland flat field. For instance you are missing how many offenders there were and how many have been arrested, the field just details whether or a single arrest has been made.  There are many arrests that never see the court system because they are kicked before entering the system, this data is not currently available.  

As for the speedy justice issue we would need a significant increase in the amount of data from the courts to determine this and to determine the causes.  I am not sure a quick end to a case is good thing.  Also, there are significant interests by the defense to drag on cases as long as they can hoping the time drives either the victim or the witnesses to stop participating.  We would have to have access to exactly who filed the motions for extensions and why, then we would have to make a qualitative judgement ab out the legitimacy of those requests.  I am no where near ready to do such a thing.  

On Dec 17, 2014, at 1:04 PM, Forest Gregg notifications@github.com wrote:

> That's interesting about the arrest date. 
> 
> On Wed, Dec 17, 2014 at 12:50 PM, Geoffrey Hing notifications@github.com 
> wrote: 
> 
> > Oh, also from this data alone , I don't think we can map to police 
> > district because the address is the convicted person's stated home address, 
> > not the address of the crime/arrest. 
> > 
> > — 
> > Reply to this email directly or view it on GitHub 
> > https://github.com/sc3/cook-convictions-data/issues/20#issuecomment-67371631 
> > . 
> 
> ## 
> 
> 773.888.2718
> —
> Reply to this email directly or view it on GitHub.

— 
We have moved!

Tracy Siska | Executive Director
Chicago Justice Project | 811 S. Bishop St. Suite 2R | Chicago, IL 60607
Ph. (312) 971-6745

tsiska@chicagojustice.org | www.chicagojustice.org
Twitter: CJPJustProj | Facebook: Become a Fan
